src/yz_index.erl

```diff
@@ -189,10 +189,18 @@ local_create(Name) ->
                     lager:info("Created index ~s with schema ~s",
                                [Name, SchemaName]),
                     ok;
+                {error, exists} ->
+                    lager:info("Index ~s already exists in Solr, "
+                               "but not in Riak metadata",
+                               [Name]);
                 {error, Err} ->
                     lager:error("Couldn't create index ~s: ~p", [Name, Err])
             end,
             ok;
+        {error, notfound} ->
+            lager:error("Couldn't create index ~s because the schema ~s isn't found",
+                        [Name, SchemaName]),
+            ok;
         {error, Reason} ->
```

Comment:
Dialyzer's just complaining about not matching this case anymore. Covered by `{'error','notfound'} | {'ok',binary()}`.